wouso/games/teamquest/models.py

```diff
@@ -14,7 +14,10 @@ def is_head(self):
 
 
 class TeamQuest(models.Model):
-	pass
+    start_time = models.DateTimeField(default=None, blank=True, null=True)
```

Comment:
`null` and `blank` should be `False`, there should be no `default`. One should provide explicit times when creating a quest.